### PR TITLE
Landing page template

### LINKED ISSUE #16: Create template for goals list page
- [x] Welcome message.
- [x] List title / question.
- [x] Add button.
- [x] List of goals.
- [x] Style the goals in the list  

Designing page for user story #3.

### PR DESCRIPTION
**Pull request by emilyb7**
Related to #25, #24 

Fixes #16

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 ./public/.sass-cache
+public/styles/.sass-cache/*
+.DS_Store
+public/.DS_Store
```
Comment (des-des) [inline .gitignore]: can use **.DS_Store?

File: public/styles/style.css [hunk 2]
```diff
@@ -1,3 +1,106 @@
+html {
```
Comment (des-des) [inline public/styles/style.css]: This is generated code?
if so
1. it makes qa harder
2. it grows the size of .git file
3. sometimes can result in nasty merge conflicts.

Atm there is no description of build step in readme..

File: public/styles/style.scss [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+
+.goal-item-img-container {
+  height: 70px;
+  width: 70px;
```
Comment (des-des) [inline public/styles/style.scss]: Are these `70px` representing the same quantities? With sass you can define them in a single var

File: public/styles/style.css (hunk 2 above)
Comment (emilyb7) [inline public/styles/style.css]: @des-des we don't know what you mean, we haven't touched style.css, it's being compiled with sass...

File: public/styles/style.scss (hunk 3 above)
Comment (emilyb7) [inline public/styles/style.scss]: yes, good point...

File: public/styles/style.css (hunk 2 above)
Comment (des-des) [inline public/styles/style.css]: Yes, but you have committed it to the code base. The idea is that you only commit source code..

eg if using gh-pages, you would only commit this file to gh-pages branch, and would not add it to master.. Does that make sense?

Comment (emilyb7) [inline public/styles/style.css]: i think so... is this better now?

Comment (emilyb7) [inline public/styles/style.css]: just realised how i've always done this very wrong

File: public/styles/style.css [hunk 4]
```diff
@@ -1,3 +1,43 @@
+/*
+Error: Invalid CSS after "$goal-box-size": expected ":", was "html {"
```
Comment (des-des) [inline public/styles/style.css]: Urm, actually I think we still have a problem..

Comment (emilyb7) [inline public/styles/style.css]: oops. now?

**Comment by des-des** [comment]
@emilyb7 Happy to merge this with compiled css included in the pull, as long as my point makes sense to you?

**Comment by emilyb7** [comment]
@des-des i've removed the file now!

**Comment by des-des** [comment]
:tada:

**Pull request merged**